### ISSUE TITLE
Won't compile with Xcode 8 GM

### ISSUE THREAD

**Issue by dickwu** [ReactiveX/RxSwift#870]
You're probably aware of this but RxSwift won't compile on Xcode 8 GM.

**Comment by linqingmo** [comment]
It seems we need to add @escaping in front of all closures in init(...).

**Comment by ivanruizscm** [comment]
any updates on this?

**Comment by kzaher** [comment]
https://github.com/ReactiveX/RxSwift/pull/869

**Comment by RubenSandwich** [comment]
Great work guys!

When this work will be released?

**Comment by kzaher** [comment]
@RubenSandwich 
3.0.0-beta.1 pre-release with these changes probably on Sunday/Mondayish

3.0.0 release is expected in 4 weeks. We'll have at least one beta and one rc before.

**Comment by kzaher** [comment]
This issue is still present with Xcode 8 GM and iOS 10.
https://github.com/ReactiveX/RxSwift/issues/842

Is it possible Apple has this bug, or are we doing something wrong?

**Comment by juliengdt** [comment]
use 

```
pod 'RxSwift', :git => 'https://github.com/ReactiveX/RxSwift.git'
```

for the moment, last release is coming in some days

**Comment by kzaher** [comment]
Hi @dickwu ,

just released 3.0.0-beta.1. This should work now.
